src/docfx/restore/RestoreFile.cs

```diff
@@ -77,6 +77,8 @@ public static IEnumerable<string> GetFileReferences(this Config config)
 
             yield return config.MonikerDefinition;
 
+            yield return config.DependencyLock;
```

Comment:
consider define this as local file only

Reply:
We can do it in another PR, will add a todo here.